### PR TITLE
Initial CHANGELOG.md

### PR DESCRIPTION
**Pull request by teskey**
Hey there! So you want to contribute to a Firebase SDK? 
Before you file this pull request, please read these guidelines:

### Discussion

  * Read the contribution guidelines (CONTRIBUTING.md).
  * If this has been discussed in an issue, make sure to link to the issue here. 
    If not, go file an issue about this **before creating a pull request** to discuss.

### Testing

  * Make sure all existing tests in the repository pass after your change.
  * If you fixed a bug or added a feature, add a new test to cover your code.

### API Changes

  * At this time we cannot accept changes that affect the public API.  If you'd like to help 
    us make Firebase APIs better, please propose your change in an issue so that we 
    can discuss it together.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
...
+  [`bucket()`](https://firebase.google.com/docs/reference/admin/node/admin.storage.Storage#bucket) method
+  in the Storage API.
+
+# v5.2.1
+
+- [changed] A bug in the TypeScript type declarations that come bundled with the
+  SDK (`index.d.ts`) has been fixed.
+
+# v5.2.0
+- [added] A new [{{firebase_storage}} API](https://firebase.google.com/docs/reference/admin/node/admin.storage)
```
Comment (hiranya911) [inline CHANGELOG.md]: Replace {{firebase_storage}} with Firebase Storage.

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+  available. This API lets you manage your users without using their existing
+  credentials, and without worrying about client-side rate limiting. The new
+  methods included in this API are
+  [`getUser()`](https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#getUser),
+  [`getUserByEmail()`](https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#getUserByEmail),
+  [`createUser()`](https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#createUser),
+  [`updateUser()`](https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#updateUser), and
+  [`deleteUser()`](https://firebase.google.com/docs/reference/admin/node/admin.auth.Auth#deleteUser). See
+  [Manage Users](https://firebase.google.com/docs/auth/admin/manage-users) for more details.
+- {[changed] The
```
Comment (hiranya911) [inline CHANGELOG.md]: Extra '{'

File: CHANGELOG.md (hunk 1 above)
Comment (teskey) [reply CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (teskey) [reply CHANGELOG.md]: Done

**Pull request merged**